tests/integration/components/project-members-test.js

```diff
@@ -28,7 +28,7 @@ test('it renders an item for each member in the list', function(assert) {
   }
 
   this.set('users', mockUsers);
-  page.render(hbs`{{project-members users=users}}`);
+  page.render(hbs`{{project-users users=users}}`);
 
   assert.equal(page.memberCount, 3, 'The correct number of members are rendered');
   assert.ok(page.members(0).imageSource.indexOf('image_1.png') > -1, 'The correct photo renders');
```

Comment:
I would change this and the page objects to be `user` and `users` instead of `member` and `members`, as well.